csharp/NFT/NFTContract.Helper.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using Neo.SmartContract.Framework;
+using Neo.SmartContract.Framework.Services.Neo;
+
+namespace NFT
+{
+    partial class NFTContract
+    {
+        /// <summary>
+        /// Convert To BigInteger Safely
+        /// </summary>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        private static BigInteger ToBigInteger(ByteString value) => value is null ? 0 : (BigInteger)value;
```

Comment:
Reuse this method? or move this to SDK? https://github.com/neo-project/neo-devpack-dotnet/blob/522a12fcf5dbe6e4d21b1f2c9082bd30d226ddae/src/Neo.SmartContract.Framework/Helper.cs#L33